Dockerfile

```diff
@@ -1,53 +1,63 @@
-FROM debian:jessie
+FROM erlang:20.3 AS build-env
 
-MAINTAINER Octavo Labs AG info@vernemq.com
+WORKDIR /tmp/vernemq-build
 
-RUN apt-get update && apt-get install -y \
-    libssl-dev \
-    logrotate \
-    sudo \
-    curl \
-    jq \
-    && rm -rf /var/lib/apt/lists/*
-
-ENV VERNEMQ_VERSION 1.6.1
+ARG VERNEMQ_VERSION=1.6.0
+ARG TARGET rel
+ARG VERNEMQ_REPO=https://github.com/vernemq/vernemq.git
 
 # Defaults
 ENV DOCKER_VERNEMQ_KUBERNETES_APP_LABEL vernemq
 ENV DOCKER_VERNEMQ_LOG__CONSOLE console
 
-ADD https://bintray.com/artifact/download/erlio/vernemq/deb/jessie/vernemq_$VERNEMQ_VERSION-1_amd64.deb /tmp/vernemq.deb
+RUN \
+    apt-get update \
+    && apt-get -y install build-essential git libssl-dev  \
+	&& rm -rf /var/lib/apt/lists/* \
```

Comment:
I think this is redundant, as you are in the builder image, so you don't really need to clean anything up.